src/coreclr/src/vm/jithelpers.cpp

```diff
@@ -5041,6 +5041,59 @@ Thread * __stdcall JIT_InitPInvokeFrame(InlinedCallFrame *pFrame, PTR_VOID StubS
 EXTERN_C void JIT_PInvokeBegin(InlinedCallFrame* pFrame);
 EXTERN_C void JIT_PInvokeEnd(InlinedCallFrame* pFrame);
 
+// Forward declaration
+EXTERN_C void STDCALL ReversePInvokeBadTransition();
+
+// This is a slower version of the reverse PInvoke enter function.
+NOINLINE static void JIT_ReversePInvokeEnterRare(ReversePInvokeFrame* frame)
+{
+    _ASSERTE(frame != NULL);
+
+    Thread* thread = GetThreadNULLOk();
+    if (thread == NULL)
+        CREATETHREAD_IF_NULL_FAILFAST(thread, W("Failed to setup new thread during reverse Platform Invoke"));
```

Comment:
```suggestion
        CREATETHREAD_IF_NULL_FAILFAST(thread, W("Failed to setup new thread during reverse PInvoke"));
```

We typically use PInvoke or P/Invoke, even in user facing strings. I have searched for "platform invoke" under coreclr and it is used just 3 times in an internal comment. "PInvoke" is used 2000 times, including many occurrences in public facing strings like error messages.